### PR TITLE
Shipfile ut

### PR DESCRIPTION
**Pull request by lewisrenfrew**
- still need to finalise the html -> pdf template and add it in
- there's probably too much going on in this PR to really digest in a quick review. I can show and tell all this stuff again now that it's less abstract

### CODE REVIEW

File: src/Domain.LinnApps/Domain.LinnApps.csproj [hunk 1]
```diff
@@ -8,10 +8,13 @@
 
   <ItemGroup>
     <PackageReference Include="Linn.Common.Authorisation" Version="1.0.0" />
+    <PackageReference Include="Linn.Common.Configuration" Version="2.0.0" />
     <PackageReference Include="Linn.Common.Domain" Version="1.0.0" />
     <PackageReference Include="Linn.Common.Domain.LinnApps" Version="1.0.0" />
     <PackageReference Include="Linn.Common.Persistence" Version="2.1.1" />
     <PackageReference Include="Linn.Common.Reporting" Version="1.4.0" />
+    <PackageReference Include="System.Drawing.Common" Version="5.0.2" />
+    <PackageReference Include="System.Text.Encoding.CodePages" Version="5.0.0" />
```
Comment (lewisrenfrew) [inline src/Domain.LinnApps/Domain.LinnApps.csproj:17]: note to self: are these actually used?

File: src/Service.Host/client/src/middleware/receiveDeletedConsignmentShipfile.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import * as actionTypes from '../actions';
+import consignmentShipfilesActions from '../actions/consignmentShipfilesActions';
+
+export default ({ dispatch }) => next => action => {
+    const result = next(action);
+
+    switch (action.type) {
+        case actionTypes.consignmentShipfileActionTypes.RECEIVE_DELETED_CONSIGNMENT_SHIPFILE:
+            console.log('we caught you');
```
Comment (richardiphillips) [inline src/Service.Host/client/src/middleware/receiveDeletedConsignmentShipfile.js:9]: console logging

File: src/Service/ResponseProcessors/ConsignmentShipfileResponseProcessor.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿namespace Linn.Stores.Service.ResponseProcessors
+{
+    using Linn.Common.Facade;
+    using Linn.Common.Nancy.Facade;
+    using Linn.Stores.Domain.LinnApps;
+
+    public class ConsignmentShipfileResponseProcessor : JsonResponseProcessor<ConsignmentShipfile>
+    {
+        public ConsignmentShipfileResponseProcessor(IResourceBuilder<ConsignmentShipfile> resourceBuilder)
+            : base(resourceBuilder, "consignmnet-shipfile-1")
```
Comment (richardiphillips) [inline src/Service/ResponseProcessors/ConsignmentShipfileResponseProcessor.cs:10]: spelling 
- should be ,1 too ?

**Review by richardiphillips — APPROVED** [review]
looks good

File: src/IoC/LoggingModule.cs [hunk 4]
```diff
@@ -10,13 +10,8 @@ public class LoggingModule : Module
     {
         protected override void Load(ContainerBuilder builder)
         {
-#if DEBUG
+
```
[the remark this answers is not in the document]
Comment (lewisrenfrew) [reply src/IoC/LoggingModule.cs:13]: uh oh, that wasn't deliberate. Good catch

**Pull request merged**